homeassistant/components/lg_soundbar/media_player.py

```diff
@@ -53,8 +53,13 @@ def __init__(self, discovery_info):
         self._woofer_volume_max = 0
         self._bass = 0
         self._treble = 0
+        self._device = None
 
-        self._device = temescal.temescal(host, port=port, callback=self.handle_event)
+    async def async_added_to_hass(self):
+        """Register the callback after hass is ready for it."""
+        self._device = temescal.temescal(
+            self._host, port=self._port, callback=self.handle_event
+        )
         self.update()
```

Comment:
Does this do blocking I/O? Then we need to use the executor via `hass.async_add_executor_job`.

Reply:
Hmm yes the connect() is blocking. Let me fix that up.